travis/setup_xspec_ds9.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# Tarballs to fetch
+DS9_SITE=http://ds9.si.edu/download/centos6/;
+XPA_SITE=http://ds9.si.edu/download/centos6/;
+DS9_TAR=ds9.centos6.7.5.tar.gz;
+XPA_TAR=xpa.centos6.2.1.17.tar.gz;
+
+# Fetch them
+wget --quiet $DS9_SITE$DS9_TAR;
+wget --quiet $XPA_SITE$XPA_TAR;
+
+# untar them
+THIS_DIR=$(pwd);
+cd $MINICONDA/bin; tar xf $THIS_DIR/$DS9_TAR; tar xf $THIS_DIR/$XPA_TAR; cd -;
+
+# install build dependencies
+sudo apt-get update
+sudo apt-get install -qq libwcs4 wcslib-dev libx11-dev libsm-dev libxrender-dev;
+
+# Some relevant environment variables
+export HEADAS=$MINICONDA/envs/build/Xspec/spectral;
+export LD_LIBRARY_PATH=$MINICONDA/envs/build/Xspec/x86_64-unknown-linux-gnu-libc2.15-0/lib/;
```

Comment:
I've never had to set LD_LIBRARY_PATH to this to get Sherpa to work, but maybe I've been lucky (possibly because I've tended to build a full XSPEC installation rather than a models-only build, and maybe the installation is subtly-different for these two approaches).

Reply:
I don't usually do that either, but it depends on the setups, and I didn't question the previous setup. I'll try removing that line and see what happens.